certified-connectors/QPP NG Integration Hub/apiDefinition.swagger.json

```diff
@@ -1793,20 +1868,21 @@
         }
       }
     },
-	"/api/v1/attributeService/getAllAttributes": {
+    "/api/v1/attributeService/getAllAttributes": {
       "get": {
         "summary": "getAllAttributes",
-        "description": "getAllAttributes",
+        "description": "Get All Attributes from QPP NG Enviroment. For more information Check the github documentation.",
```

Comment:
Please correct the spelling "enviroment"

Reply:
Done